src/libmongoc/tests/unified/operation.c

```diff
@@ -28,6 +28,7 @@ typedef struct {
    bson_t *arguments;
    bson_t *expect_error;
    bson_val_t *expect_result;
+   bool *ignore_result_and_error;
```

Comment:
Just noticed this is a pointer (based on how `bson_parser_bool_optional` works). Do these fields get freed anywhere? I don't see any other reference to this field beyond parsing and reading it, but I assume `bson_parser_destroy_with_parsed_fields` takes care of this?

Reply:
In this case, `bson_parser_entry_destroy` handles freeing the pointer: https://github.com/mongodb/mongo-c-driver/blob/d8684bed2f5d0763aceb8c03cbce1e2637bc515c/src/libmongoc/tests/bsonutil/bson-parser.c#L253..L258